FeedAPIChallenge/Feed API/FeedImageMapper.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FeedImageMapper.swift
+//  FeedAPIChallenge
+//
+//  Created by Mauro Worobiej on 05/09/2021.
+//  Copyright © 2021 Essential Developer Ltd. All rights reserved.
+//
+
+import Foundation
+
+class FeedImageMapper {
+	private struct Root: Decodable {
+		let items: [ItemModel]
+
+		var feedImage: [FeedImage] {
```

Comment:
Consider naming this as a plural to indicate it is a collection as opposed to a single value.
```suggestion
		var feedImages: [FeedImage] {
```

Reply:
Nice catch!